toolkit/tools/imagecustomizer/main.go

```diff
@@ -18,17 +18,18 @@ import (
 var (
 	app = kingpin.New("imagecustomizer", "Customizes a pre-built CBL-Mariner image")
 
-	buildDir                 = app.Flag("build-dir", "Directory to run build out of.").Required().String()
-	imageFile                = app.Flag("image-file", "Path of the base CBL-Mariner image which the customization will be applied to.").Required().String()
-	outputImageFile          = app.Flag("output-image-file", "Path to write the customized image to.").Required().String()
-	outputImageFormat        = app.Flag("output-image-format", "Format of output image. Supported: vhd, vhdx, qcow2, raw.").Required().Enum("vhd", "vhdx", "qcow2", "raw")
-	configFile               = app.Flag("config-file", "Path of the image customization config file.").Required().String()
-	rpmSources               = app.Flag("rpm-source", "Path to a RPM repo config file or a directory containing RPMs.").Strings()
-	disableBaseImageRpmRepos = app.Flag("disable-base-image-rpm-repos", "Disable the base image's RPM repos as an RPM source").Bool()
-	logFile                  = exe.LogFileFlag(app)
-	logLevel                 = exe.LogLevelFlag(app)
-	profFlags                = exe.SetupProfileFlags(app)
-	timestampFile            = app.Flag("timestamp-file", "File that stores timestamps for this program.").String()
+	buildDir                    = app.Flag("build-dir", "Directory to run build out of.").Required().String()
+	imageFile                   = app.Flag("image-file", "Path of the base CBL-Mariner image which the customization will be applied to.").Required().String()
+	outputImageFile             = app.Flag("output-image-file", "Path to write the customized image to.").Required().String()
+	outputImageFormat           = app.Flag("output-image-format", "Format of output image. Supported: vhd, vhdx, qcow2, raw.").Required().Enum("vhd", "vhdx", "qcow2", "raw")
```

Comment:
You can make this optional now. But you should enforce that either `--output-image-format` or `--output-split-partitions-format` is specified or both.

Reply:
Got it; will be handled in a future PR - requires several small changes since currently the code is written with the assumption that outputImageFormat is required.